### ISSUE TITLE
Update Legends of Learning icons for Build an Atom

### ISSUE THREAD

**Issue by samreid** [phetsims/build-an-atom#171]
See #162 

@amanda-phet delivered the icons to https://drive.google.com/drive/folders/0B6CMwxdP0NGYZkN5WHNXOHRFQTQ

**Comment by samreid** [comment]
This is on hold until we can submit separate entries for our screens.

**Comment by samreid** [comment]
@arouinfar and @amanda-phet what is the status with respect to LOL?  Can we submit things yet?

**Comment by amanda-phet** [comment]
The icons were uploaded on August 10th to this folder: https://drive.google.com/drive/folders/0B3v_ZP9Kq36CZ2JUc09iODItbTA 

Do you want me to also add them to assets here in GitHub?0

**Comment by arouinfar** [comment]
All Legends of Learning materials are in the PhET Partnerships > Deliverables > Legends of Learning folder on Drive. KP has shared the[ LoL Simulations](https://docs.google.com/spreadsheets/d/1umIAmhn89WN1nzcHKhYJcv-n3Oj6ps1wITc-CjWYytE/edit#gid=0) spreadsheet with the folks at LoL that includes the sim descriptions, links to the sim, and links to the icons (stored on Drive).  The “Ready_To_Integrate” tab contains the sims that we have delivered to LoL. Build an Atom is included in that tab, so there's nothing else that needs to be done. 

I think it would be helpful if everyone involved in the LoL partnership had a 15-20 minute slot at the next design meeting to iron out the process and make sure everyone is in the loop. I've created a reminder to request time for LoL at next week's design meeting, so I think we can close this issue @samreid.
